.ci.yaml

```diff
@@ -84,6 +84,29 @@ targets:
       - cipd_packages/codesign/**
       - .ci.yaml
 
+  - name: Mac ruby
+    bringup: true
+    recipe: cocoon/cipd
+    properties:
+      script: cipd_packages/ruby/build.sh
+      cipd_name: flutter/ruby/mac-amd64
+      device_type: iOS
+    runIf:
+      - cipd_packages/ruby/**
+      - .ci.yaml
+
+  - name: Mac arm_ruby
```

Comment:
We should define a new platform: Mac_arm to build an arm versioned ruby.

Reply:
Not sure I follow what would be the benefit of creating a new platform? the cpu dimension will achieve the same purpose.

Reply:
This is to keep consistent with other repos. This is not a big issue for Cocoon for now, which has a limited number of targets. If we have more targets in the future, we have to update the target level `cpu` for each of the targets. With a platform entry, you just need to update a single place.